job_server/package.json

```diff
@@ -1,13 +1,18 @@
 {
-    "name": "server",
+    "name": "internado-job-server",
     "version": "0.0.1",
     "description": "Find personalized internship opportunities based on your skills and interests",
     "main": "server.js",
     "scripts": {
-        "test": "echo \"Error: no test specified\" && exit 1",
-        "build": "tsc -p . && tslint -p tsconfig.json -c tslint.json",
-        "start": "npm run build && ts-node server.ts",
-        "clean": "rimraf **/*.js **/*.js.map node_modules"
+        "build": "npm run clean && gulp copy && tsc -p .",
```

Comment:
Can you send me the error log ?

Reply:
I reworote the npm scripts so that the dev doesnt have to build and then start the server. Right now,
- npm start : builds the ts files to js & copies swagger.json , runs linter , starts server 
- npm run lint : runs the linter
- npm run build : builds ts to js and copies swagger.json (using gulp)

Also did you try out 
- npm run dev ? It watches for changes and auto builds and starts the server for you

Reply:
When running `docker-compose build`, it runs for a while and seems to fail when running the gulp command. Here is the log of the error: 
```
[01:48:46] No gulpfile found
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! internado-job-server@0.0.1 build: `npm run clean && gulp copy && tsc -p .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the internado-job-server@0.0.1 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-03-03T01_48_46_242Z-debug.log
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! internado-job-server@0.0.1 prepare: `npm run build && npm run lint`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the internado-job-server@0.0.1 prepare script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2019-03-03T01_48_46_322Z-debug.log
ERROR: Service 'job_server' failed to build: The command '/bin/sh -c npm run prepare' returned a non-zero code: 1
```
Is there a gulpfile that needs to be added to the folder?